scipy/fft/_basic.py

```diff
@@ -183,6 +192,9 @@ def ifft(x, n=None, axis=-1, norm=None, overwrite_x=False):
     overwrite_x : bool, optional
         If True, the contents of `x` can be destroyed; the default is False.
         See :func:`fft` for more details.
+    workers : int, optional
+        Maximum number of workers to use for parallel computation, or 0 for
+        unlimited. Must be -1 or >0. See `fft` for more details.
```

Comment:
I would rather mirror what `joblib` does here where < 0 means `n_cpus + 1 + n_workers` a) for consistency and b) so you can for instance also do `workers=-2` to mean "all my CPUs except one".